dodo.py

```diff
@@ -64,115 +114,128 @@ def task_po():
     """
 
     def process_locales():
+        subprocess.run(
+            [
+                "python",
+                "bin/i18n/prop2po.py",
+                "--help"
+            ]
+        )
         for current_locale in LOCALES:
 
-            LOC_PATH = "tabcmd/locales/" + current_locale
-            for file in glob.glob(LOC_PATH + "/*.properties"):
-                basename = os.path.basename(file).split(".")[0]
-                print("processing", basename)
-                result = subprocess.run(
-                    [
-                        "python",
-                        "bin/i18n/prop2po.py",
-                        "--encoding",
-                        "utf-8",  # for the .po header
-                        "--language",
-                        current_locale,  # for the .po header
-                        LOC_PATH + "/" + basename + ".properties",
-                        LOC_PATH + "/LC_MESSAGES/" + basename + ".po",
-                    ]
-                )
-                print("\n", result)
-                # print("stdout:", result.stdout)
+            LOC_PATH = os.path.join("tabcmd", "locales", current_locale, "LC_MESSAGES")
+            PROPS_FILE = os.path.join(LOC_PATH, "transcoded.properties")
+            PO_FILE = os.path.join(LOC_PATH, "tabcmd.po")
+            LOG_FILE = os.path.join(LOC_PATH, "prop2po.out")
+            with open(LOG_FILE, "w+", encoding="utf-8") as logfile:
+                try:
+                    result = subprocess.run(
+                        [
+                            "python",
+                            "bin/i18n/prop2po.py",
+                            "--encoding",
+                            "utf-8",  # for the .po header
+                            "--language",
+                            current_locale,  # for the .po header
+                            "--project",
+                            "Tabcmd 2",
+                            "--copyright",
+                            "TABLEAU SOFTWARE, LLC, A SALESFORCE COMPANY. ALL RIGHTS RESERVED",
```

Comment:
Does this copyright need to match the new guidelines? i.e. `©2024 Salesforce, Inc.`